storage/src/persistent/mod.rs

```diff
@@ -31,6 +32,8 @@ pub mod database;
 pub mod schema;
 pub mod sequence;
 
+const PRESERVE_CYCLE_COUNT: usize = 7;
```

Comment:
If we have a source for this number it would be really good to link it here.

Reply:
@mat13mn 
please add here TODO with jira for real integartion to GC trigger